pydarn/plotting/rtp.py

```diff
@@ -0,0 +1,438 @@
+            default: True
+        color_bar_label: str
+            the label that appears next to the color bar
+            default: ''
+            Certain standard parameters have pre-set labels:
+            elevation: 'elevation $degrees$'
+            power: 'signal to noise $dB$'
+            spectral width: 'spectral width $m/s$'
+            velocity: 'velocity $m/s$'
+        color_map: str
+            matplotlib colour map
+            https://matplotlib.org/tutorials/colors/colormaps.html
+            default: jet
+            note: to reverse the color just add _r to the string name
+        color_norm: matplotlib.colors.Normalization object
+            This object use dependency injection to use any normalization
+            method with the zmin and zmax.
+            defualt: colors.Normalization()
+        ax: matplotlib.axes
+            axes object for another way of plotting
+            default: None
+        boundary: (int, int)
+            min and max values to include in the plot and set for normalization
+            of the color map.
+            default: None
+                - the min and max values in the data are used instead
+        max_array_filter : dict
+            dictionary that contains the key parameter names and the values to
+            compare against. Will filter out any data points
+            that is above this value.
+        min_array_filter : dict
+            dictionary that contains the key parameter names and the value to
+            compare against. Will filter out any data points that is
+            below this value.
+        max_scalar_filter : dict
+            dictionary that contains the key parameter names and the values to
+            compare against. Will filter out data sections that is
+            above this value.
+        min_scalar_filter : dict
+            dictionary that contains the key parameter names and the value to
+            compare against. Will filter out data sections
+            that is below this value.
+        equal_scalar_filter : dict
+            dictionary that contains the key parameter names and the value to
+            compare against. Will filter out data sections
+            that is does not equal the value.
+
+
+        Raises
+        ------
+        RTPUnknownParameterError
+        RTPInocrrectParameterPlotError
+        RTPNoDataFoundError
+
+        Returns
+        -------
+        im: matplotlib.pyplot.pcolormesh
+            matplotlib object from pcolormesh
+        cb: matplotlib.colorbar
+            matplotlib color bar
+        cmap: matplotlib.cm
+            matplotlib color map object
+        """
+        # Settings
+        settings = {'ground_scatter': False,
+                    'channel': 'all',
+                    'date_fmt': "%y/%m/%d\n%H:%M",
+                    'color_bar': True,
+                    'color_bar_label': '',
+                    'color_map': 'jet',
```

Comment:
Let's be bold and change the default colormap.  If we want to keep the red/blue we can use Spectral_r or seismic_r